pyqtgraph/widgets/SpinBox.py

```diff
@@ -132,8 +132,10 @@ def setOpts(self, **opts):
         siPrefix       (bool) If True, then an SI prefix is automatically prepended
                        to the units and the value is scaled accordingly. For example,
                        if value=0.003 and suffix='V', then the SpinBox will display
-                       "300 mV" (but a call to SpinBox.value will still return 0.003). Default
-                       is False.
+                       "300 mV" (but a call to SpinBox.value will still return 0.003). In case
+                       the value represents a dimensionless quantity that might span many
+                       orders of magnitude, such as a Reynold's number, an SI
```

Comment:
slight nitpick, the guy's name was Reynolds, so it's "a Reynolds number", no apostrophe.

Reply:
that is as slight of a nitpick as nitpicks get! 😆 I'd merge a PR making that change.

Reply:
Hey, what can I say, I am/was a fluid dynamics guy :D